praw/models/reddit/subreddit.py

```diff
@@ -2042,6 +2048,16 @@ def delete_all(self) -> List[Dict[str, Union[str, bool, Dict[str, str]]]]:
         """
         return self.update(x["user"] for x in self())
 
+    def reorder(self, newlist, flair_type):
+        """Reorder a list of flairs.
+        :param newlist: A list of Flair ID's.
+        :param flair_type : The flairs types, either USER_FLAIR or LINK_FLAIR
+        """
+        if flair_type not in ["USER_FLAIR", "LINK_FLAIR"]:
+                raise TypeError("'flair_type' must be either USER_FLAIR or LINK_FLAIR")
+        url = API_PATH["flairreorder"].format(subreddit=self.subreddit)
+        self.subreddit._reddit.patch(url+"?subreddit="+self.subreddit.display_name+"&flair_type="+flair_type, json=newlist)
```

Comment:
Are you sure the arguments need to be url parameters?

Reply:
> Are you sure the arguments need to be url parameters?

Both posts I found on the subject did it this way, and it was the only method that worked for me... Not to say there isn't another way though

Reply:
> Are you sure the arguments need to be url parameters?

Got confirmation from Reddit, yes, this is how to do it.

> Run prepush.py

I got an error with this one: 
Running: pre-commit run --all-files
[WinError 2] The system cannot find the file specified
Which file is that>

> rebase to master

I did this following instructions on the web, and it doesn't seem to have made any difference? I've pushed my latest changes either way

Reply:
> I got an error with this one:
> Running: pre-commit run --all-files
> [WinError 2] The system cannot find the file specified
> Which file is that>

Did you by chance install dev dependencies? You can do this with `pip install .[dev]` in the repo root.

> Got confirmation from Reddit, yes, this is how to do it.

If that is the case, then let's add a new `params` parameter to the `patch` method. See the `get` or `post` methods for how this is utilized.